app/controllers/bookmarks_controller.rb

```diff
@@ -253,7 +253,7 @@ def update
 
     if @bookmark.update(bookmark_params) && errors.empty?
       flash[:notice] ||= ""
-      flash[:notice] = ts(" Bookmark was successfully updated. ").html_safe + flash[:notice]
+      flash[:notice] = ts("Bookmark was successfully updated.").html_safe + flash[:notice]
```

Comment:
Is stripping the trailing space here going to make the notice run into the collection notice? It looks like the `unapproved_collections` notice has a space at the front, but the `new_collections` notice doesn't.

Reply:
I was going to say it's okay, but I just double checked and the test I looked at is actually for using the "Add to Collections" button on the bookmark (which leaves off the "Bookmark was successfully updated" part), not the "Edit" button, so we should probably put it back.